config/routes.rb

```diff
@@ -18,10 +17,12 @@
   resources :volunteers do
     get :seeking_clients, on: :collection
     get :find_client, on: :member, to: 'assignments#find_client'
+    get :volunteer_hours, on: :member
     resources :journals
   end
   resources :volunteer_emails
   resources :profiles, except: [:destroy, :index]
+  resources :hours
```

Comment:
I thought this should have been a nested route too, just like journals are!?

And in that case, it would also make that second parameter in confirm_deleting() obsolete.